ci3.8.yml

```diff
@@ -31,5 +30,6 @@ dependencies:
   - pygrib=2.1.4
   - pip:
     - earthengine-api==0.1.329
+    - xarray==2023.1.0
```

Comment:
On second though: we may get weird link errors with our conda deploy if we don't include this as a conda dependency. Has this been published on conda forge yet?

Reply:
While trying to install the `xarray` package via the `conda-forge` channel, I encountered an error when running the installation in the **CI/CD pipeline**. However, the installation process worked without any issues when I tried it **locally**. Here's an error message that I received during the CI/CD run:
![image](https://github.com/google/weather-tools/assets/123355381/eb9649ee-e374-46a6-8507-ea4ed7a900d4)
 [**Link**](https://github.com/google/weather-tools/actions/runs/4829131037/jobs/8603795741)
Do you have any suggestions for resolving this issue?

Reply:
It looks like 2023.4.2 is available on conda (I think this is the latest release). How about we try upgrading to that version? 

https://github.com/conda-forge/xarray-feedstock#current-release-info

Reply:
As mentioned in the PR Description `This is the last updated version of xarray that python 3.8 supported`.

Reply:
"Hello @alxmrs, I have tested both the `local` and `docker` deployment and did not encounter any errors."

Reply:
Thanks for confirming!